projects/lucidly/index.js

```diff
@@ -1,18 +1,65 @@
-async function tvl(api) {
```

Comment:
can we retain this old code in the new adapter for historical accuracy?

Reply:
yeah

Reply:
added @waynebruce0x